### PR TITLE
Allow for a zero-byte secret file

### PR DESCRIPTION
**Pull request by glennslaven**

### CODE REVIEW

**Comment by ivan-section-io** [comment]
This changes the default operation. Documentation in README.md needs to be updated to match.

File: README.md [hunk 1]
```diff
@@ -42,8 +42,11 @@ Defaults to `varnish` if not provided.
 secret used to authenticate connections to the Varnish CLI Bridge.
 Can be specified via the `VARNISH_CLI_BRIDGE_SECRET_FILE` environment variable
 or the `-secret-file` command line argument, with the latter taking precedence.
-Defaults to `/etc/varnish/secret` if not provided. File contents must be
-identical to the secret file passed to `varnishadm` via its `-S` argument.
+If specified the ile contents must be identical to the secret file passed
+to `varnishadm` via its `-S` argument.
+
+If left blank the bridge will not send the authentication challenge and so the
+client should not specify a secret file or value either
```
Comment (jstangroome) [inline README.md]: Full stop.

File: README.md [hunk 2]
```diff
@@ -63,6 +66,9 @@ planned to be implemented.
 * `ban`
 * `ban.url` (via automatic rewriting to `ban`)
 * `ping`
+* `param.show`
```
Comment (jstangroome) [inline README.md]: For limited parameters IIRC

**Comment by jstangroome** [comment]
LGTM

**Pull request merged**